api/shape/Arc.js

```diff
@@ -97,7 +98,7 @@ const reifyArc =
       }
     }
 
-    return spiral.absolute();
+    return absolute()(spiral);
```

Comment:
How does this syntax work?

Reply:
At the api/shape level, operators are functors.

f = op(args) builds a function which waits for an input shape for context.
r = f(input) then calls it.

This allows us to pass around things like cut(1/4) and then later choose what shape to apply it to.

Here we're just saying, call absolute with no parameters on spiral.